backend/src/main/java/com/woowacourse/gongseek/article/application/ArticleService.java

```diff
@@ -48,8 +49,34 @@ public ArticleResponse findOne(AppMember appMember, Long id) {
         if (appMember.isGuest()) {
             return new ArticleResponse(article, false);
         }
-        Member member = findMember(appMember);
+        return new ArticleResponse(article, article.isAuthor(findMember(appMember)));
+    }
+
+    public ArticleUpdateResponse update(AppMember appMember, ArticleUpdateRequest articleUpdateRequest, Long id) {
```

Comment:
```suggestion
    @Transactional
    public ArticleUpdateResponse update(AppMember appMember, ArticleUpdateRequest articleUpdateRequest, Long id) {
```
트랜잭션 어노테이션을 붙여주세요. ⚡